package.json

```diff
@@ -201,6 +202,7 @@
     "@babel/eslint-plugin": "^7.17.7",
     "@babel/plugin-proposal-optional-chaining": "^7.21.0",
     "@babel/preset-env": "^7.22.4",
+    "@esbuild/linux-x64": "^0.21.5",
```

Comment:
@smallst this line is not really cross-platform, doesn't work on Mac. Do we need it?